docs/resources/client/i18n.md

```diff
@@ -120,6 +120,31 @@ Some places in Minecraft offer you helper methods to get a translation keys. For
 The only purpose of translation keys is for localization. Do not use them for game logic, that's what [registry names][regname] are for.
 :::
 
+### Component Translations
```

Comment:
This title is kind of misleading since what this is providing is a rich component format for translations within the lang file.